kernel_config/kernel_64_defconfig

```diff
@@ -1554,6 +1554,7 @@ CONFIG_NVME_TARGET=y
 CONFIG_MISC_RTSX=y
 CONFIG_UID_SYS_STATS=y
 # CONFIG_UID_SYS_STATS_DEBUG is not set
+CONFIG_BT_SCOHCI=m
```

Comment:
this should also go as diff config as the corresponding driver is also in diff config